usagov/usagov-2021
# USAGOV-1852-wizard-anchors: Use anchor links to direct users to content of wizard pages

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1852

## Description

Add anchor links to the urls of next/previous and start over links on wizard pages.

## Type of Changes

- [ X ] Frontend (Twig, Sass, JS)
![image](https://github.com/user-attachments/assets/30ac0c04-2d01-4767-91f4-8e24891721dd)

## Testing Instructions


1. Be logged out.
2. Navigate to any wizard page.
3. Open devtools and check the console.
4. There are no console errors.
5. Click the "next" button.
6. The "next" page loads as expected, and the main content area is in visual focus, just as it is in production in the current scams wizard.
7. Check the console. There are no console areas.
8. Repeat steps 6-7 with the "previous" and "start over" button/link.
9. You get the expected results from step 6.
10. Repeat the steps in a couple of mobile viewports.
11. The main content area comes into focus each time with the <h1> content clearly visible.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
12. Find the commit of this pull request.
13. Build and deploy the changes.
14. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
